### PR TITLE
3536 validation ui tests

### PR DESCRIPTION
**Pull request by alibombaye**

### CODE REVIEW

File: app/pages/neworg/createorg/ui.test.js [hunk 1]
```diff
@@ -224,3 +224,22 @@ test('should navigate to confirmation page when save button is clicked with no e
...
+
+  await pageSetup(t, true);
+  await t.navigateTo(pageUrl);
+
+  const saveButton = Selector('[data-test-id="save-button"] button');
+
+  await t
+    .expect(saveButton.exists).ok()
+    .click(saveButton)
+    .expect(getLocation()).eql('http://localhost:1234/admin/organisations/find/select/create/error?ods=X01&errors=OrganisationAlreadyExists%25AnotherError');
```
Comment (kat1906) [inline app/pages/neworg/createorg/ui.test.js:244]: what is the 25 for if you only joined with %?

File: app/pages/neworg/findorg/controller.js [hunk 2]
```diff
@@ -9,8 +9,8 @@ export const getFindOrgByOds = async ({ odsCode, accessToken }) => {
 
     return { success: true };
   } catch (err) {
-    if (err.status === 404 || err.status === 406) {
-      return { success: false, errorStatus: err.status };
+    if (err.response.status === 404 || err.response.status === 406) {
```
Comment (kat1906) [inline app/pages/neworg/findorg/controller.js:12]: ohhhhhh

**Comment by kat1906** [comment]
😍

**Comment by kat1906** [comment]
thanks for doing this!

File: app/pages/neworg/createorg/ui.test.js (hunk 1 above)
Comment (alibombaye) [reply app/pages/neworg/createorg/ui.test.js:244]: oh yeah I wanted to chat to you about this. For whatever reason the `+` doesn't work when passing it in via a query string. When it comes through to the route it's replaced with just a blank space. So I've had to replace it with a `%` which shows up as `%25` in the url but in the route it's just a `%` 🤷 - Not sure if that is the right way of doing it.

Comment (kat1906) [reply app/pages/neworg/createorg/ui.test.js:244]: not sure - I used `+` in capabilities selector and it works ok, just looking now

**Pull request merged**